src/aspire/image/__init__.py

```diff
@@ -157,6 +157,9 @@ def __mul__(self, other):
 
         return Image(self.data * other)
 
+    def __neg__(self):
```

Comment:
Why is this showing up in the diff? (Also for `filters.py` below.)

Reply:
Implement the `-image` operation to reverse the sign of images for convenience such as  `sim2 = ArrayImageSource(-imgs1)` in the `testInvertContrast` function.   in line 61, I use `ScalarFilter(dim=2, value=1) for d in np.linspace(1.5e4, 2.5e4, 7)` to generate identity CTF filters which need to return a `ScaledFilter` using `scale` function in `downsample` function of ImageSource class as below: 
https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/develop/src/aspire/source/__init__.py#L383
So the `scale` function in ScalarFilter can not just pass otherwise it generates an error. Removing it will use the  `scale` in base class.

Reply:
Ok sounds good.